diff --git a/node_modules/vite-react-ssg/dist/shared/vite-react-ssg.Ctg3mDmH.mjs b/node_modules/vite-react-ssg/dist/shared/vite-react-ssg.Ctg3mDmH.mjs
index e8ea990..0bf9201 100644
--- a/node_modules/vite-react-ssg/dist/shared/vite-react-ssg.Ctg3mDmH.mjs
+++ b/node_modules/vite-react-ssg/dist/shared/vite-react-ssg.Ctg3mDmH.mjs
@@ -487,7 +487,7 @@ class RemixAdapter {
     const styleCollector = getStyleCollector ? await getStyleCollector() : null;
     const helmetContext = {};
     let routerContext = null;
-    const { StaticRouterProvider, createStaticHandler, createStaticRouter } = await import('react-router-dom/server.js');
+    const { StaticRouterProvider, createStaticHandler, createStaticRouter } = await import('react-router');
     const { dataRoutes, query } = createStaticHandler([...routes], { basename: base });
     const _context = await query(request);
     if (_context instanceof Response)
